src/pages/TransactionPage/index.tsx

```diff
@@ -254,7 +263,7 @@ function UTXOOutputBox({ output, expanded }: { output: OutputFragment; expanded:
                     {trimAddress(output.to)}
                   </UTXODetailsLink>
                 );
-              }
+              } 
```

Comment:
```suggestion
              }
```

Reply:
I fixed the first two conflicts but this last one confuses me.

Reply:
It looks like you added an extra whitespace here. Suggestion removes it.

Reply:
I see, thank you.